src/vpn-api/utils/service.ts

```diff
@@ -25,6 +25,15 @@ class ServiceInstance {
 
 	constructor(private interval: number = 10 * 1000) {}
 
+	private captureException(err: Error) {
+		const tags: { [key: string]: string } = {};
+		try {
```

Comment:
Do you need this try/finally?  It wasn't there before

Reply:
Yes, if `getId` is called before the instance is registered it will thrown an error, and it possible we want to capture exceptions during registration.